BorrowBooks/features/borrowed-books/borrowed-books.css

```diff
@@ -0,0 +1,200 @@
+body {
```

Comment:
The css elements should be resizable and fit at each screen. You can check this.

Reply:
This is not priority at the moment.